packages/smallfish-plugin-env/src/index.ts

```diff
@@ -12,9 +12,11 @@ export default (api: IApi, options = {}) => {
       process.env.PORT = 7001;
     }
   }
+  api.config.disableGlobalVariables = true;
```

Comment:
only using modifyDefaultConfig can not disabled by default, because:

https://github.com/umijs/umi/pull/2340/files#diff-56ab44962993243627596dedf79d5565R289